assets/texts/fr/amoSummary.txt

```diff
@@ -1,2 +1,3 @@
-Ce module complémentaire vous donne un bon petit sélecteur d'émoticônes que vous pouvez pour en trouver 🔎 et en sélectionner. 😀
```

Comment:
"bon petit" seems a bit unusual wording for an add-on.

It's good for coffee, tea, or even maybe bed and sofas, but software programs... 😅